src/components/invite/useInvites.js

```diff
@@ -133,21 +133,25 @@ const useCollectBounty = () => {
 }
 
 const useInvited = () => {
+  const [initialized, setInitialized] = useState(false)
   const [invites, setInvites] = useState([])
   const [level, setLevel] = useState({})
   const [totalEarned, setTotalEarned] = useState(0)
 
   const updateData = async () => {
-    const user = await goodWallet.invitesContract.methods.users(goodWallet.account).call()
+    try {
+      const user = await goodWallet.invitesContract.methods.users(goodWallet.account).call()
 
-    const level = await goodWallet.invitesContract.methods.levels(user.level).call()
-    setLevel(level)
-    setTotalEarned(user.totalEarned.toNumber() / 100) //convert from wei to decimals
+      const level = await goodWallet.invitesContract.methods.levels(user.level).call()
+      setLevel(level)
+      setTotalEarned(user.totalEarned.toNumber() / 100) //convert from wei to decimals
+    } catch (e) {
+      log.error('updateData failed:', e.message, e)
+    }
   }
 
   const updateInvited = async () => {
     try {
-      updateData()
```

Comment:
actually we're not catching possible errors as we don't awaiting
the solution is to add try..catch to the each function and to add refresh function (exported from the hook) which calls them parallelly (as you wanted) via Promise.all